website/public/js/controllers/groupsCtrl.js

```diff
@@ -184,6 +184,12 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           $scope.$close();
         });
       }
```

Comment:
I put it in `MemberModalCtrl` because that's where all the other member modal functions are. It made sense to me. Is there somewhere else that would be better?